lib/util/periods.js

```diff
@@ -734,6 +735,24 @@ shaka.util.PeriodCombiner = class {
           used = false;
         }
       }
+      // In cases where we have the same video codec but different
+      // codec bases in matched representations across periods
+      // (e.g. HEVC as hev and hvc), we don't want to delete the matched stream
+      // from unusedStreamsPerPeriod until an outputStream for each
+      // video representation has been created.
+      if (outputStream.type == ContentType.VIDEO) {
```

Comment:
We also want this when using different aac profiles, for example `mp4a.40.2` for ads and `mp4a.40.29` for main content.

Reply:
@avelad Do you think this should apply to all `ContentType.AUDIO` as well?

Reply:
Yes